nightfall-deployer/contracts/State.sol

```diff
@@ -154,12 +154,13 @@ contract State is Structures, Config {
     delete proposers[proposer];
     proposers[previousAddress].nextAddress = proposers[nextAddress].thisAddress;
     proposers[nextAddress].previousAddress = proposers[previousAddress].thisAddress;
-    if(proposer == currentProposer.thisAddress) {
+   // if(proposer == currentProposer.thisAddress) {
```

Comment:
Delete the code instead of just commenting it out. As much as we can comments should be reserved for actual comments.